src/__tests__/role.js

```diff
@@ -543,3 +543,10 @@ describe('configuration', () => {
     }
   })
 })
+
+test('should get button using his text content when title is empty', () => {
```

Comment:
I wouldn't add these tests. There's certainly an argument to be made that the dependency is an implementation detail but the point of dependencies is also to cover certain functionality. Otherwise why not copy the whole test suite from dom-accessibility-api?

Reply:
Yes you're right, is a duplicate 👍